src/jit/emitxarch.cpp

```diff
@@ -1925,7 +1925,7 @@ UNATIVE_OFFSET      emitter::emitInsSizeAM(instrDesc* id, size_t code)
         // Most 16-bit operands will require a size prefix .
         // This refers to 66h size prefix override.
 
-        if  (    (attrSize == EA_2BYTE)
+        if  (    attrSize == EA_2BYTE
```

Comment:
I have a mild preference for the second, though I would collapse at least the first two lines, if not put the whole thing on a single line (since the subexpressions are short).